src/main/resources/assets/create/lang/nl_nl.json

```diff
@@ -0,0 +1,726 @@
+	"block.create.creative_crate.tooltip.summary": "Bied een eindloze vooraad blokken aan een aangeloten _Bouwtekeningkannon_",
+
+	"block.create.schematicannon.tooltip": "BOUWTEKENINGKANNON",
+	"block.create.schematicannon.tooltip.summary": "Schiet blokken om een gepostioneerde _Bouwtekening_ in de Wereld te creëeren. Gebruikt blokken uit naastgelegen inventarissen en _Buskruit_ als brandstof.",
+	"block.create.schematicannon.tooltip.control1": "Wanneer R-Geklikt",
+	"block.create.schematicannon.tooltip.action1": "Opent de _Interface_",
+
+	"block.create.schematic_table.tooltip": "BOUWTEKENING TAFEL",
+	"block.create.schematic_table.tooltip.summary": "Schrijft opgeslagen bouwtekeningen op een _Lege_ _Bouwtekening_",
+	"block.create.schematic_table.tooltip.condition1": "Wanneer voorzien van een lege Bouwtekening",
+	"block.create.schematic_table.tooltip.behaviour1": "Uploadt een gekozen bestand uit uw bouwtekeningenmap",
+
+	"block.create.shaft.tooltip": "DRIJFAS",
+	"block.create.shaft.tooltip.summary": "_Geeft_ _Rotatie_ _door_ in een rechte lijn.",
+
+	"block.create.cogwheel.tooltip": "TANDWIEL",
+	"block.create.cogwheel.tooltip.summary": "Geeft _Rotatie_ door in een rechte lijn, en naar naastgelegen _Tandwielen._",
+
+	"block.create.large_cogwheel.tooltip": "GROOT TANDWIEL",
+	"block.create.large_cogwheel.tooltip.summary": "Een grotere versie van het _TANDWIEL,_ waardoor _veranderingen_ in _Rotatiesnelheid_ mogelijk zijn indien verbonden met zijn kleinere tegenhanger.",
+
+	"block.create.encased_shaft.tooltip": "OMHULSDE DRIJFAS",
+	"block.create.encased_shaft.tooltip.summary": "_Geeft_ _Rotatie_ _door_ in een rechte lijn. Geschikt voor het doorgeven van rotatie door muren.",
+
+	"block.create.gearbox.tooltip": "VERSNELLINGSBAK",
+	"block.create.gearbox.tooltip.summary": "_Geeft_ _Rotatie_ _door_ in _Vier_ richtingen._ Keert rechte verbindingen om.",
+
+	"block.create.gearshift.tooltip": "VERSNELLINGSPOOK",
+	"block.create.gearshift.tooltip.summary": "Een verstelbare _rotatie_ _schakelaar_ voor aangesloten drijfassen.",
+	"block.create.gearshift.tooltip.condition1": "Wanneer voorzien van kracht",
+	"block.create.gearshift.tooltip.behaviour1": "_Keert_ de binnenkomende rotatie.",
+
+	"block.create.clutch.tooltip": "KOPPELING",
+	"block.create.clutch.tooltip.summary": "Een verstelbare _rotatie_ _schakelaar_ for connected shafts.",
+	"block.create.clutch.tooltip.condition1": "Wanneer voorzien van kracht",
+	"block.create.clutch.tooltip.behaviour1": "_Stopt_ met het doorgeven van kracht aan de andere kant.",
+
+	"block.create.encased_belt.tooltip": "OMHULSDE TRANSPORTBAND",
+	"block.create.encased_belt.tooltip.summary": "_Geeft_ _Rotatie_ _door_ aan zichzelf en andere _Omhulsde_ _Transportbanden._",
+	"block.create.encased_belt.tooltip.condition1": "Wanneer aangesloten",
+	"block.create.encased_belt.tooltip.behaviour1": "Attached Blocks will have the exact _same_ _rotation_ _speed_ and direction. They do not have to face the same way.",
```

Comment:
Accidentally left in the original line here

Reply:
Yhea, I need to do another pass. I playtested it a bit, there are some other mistakes here and there.